tests/cnf/ran/gitopsztp/tests/IBBF-e2e-test.go

```diff
@@ -0,0 +1,152 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/configmap"
+	"github.com/openshift-kni/eco-goinfra/pkg/hive"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	siteconfigv1alpha1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/siteconfig/v1alpha1"
+	"github.com/openshift-kni/eco-goinfra/pkg/siteconfig"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/gitdetails"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/tsparams"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"fmt"
+	"time"
+)
+
+var _ = Describe(
+	"Performing Image-Based Break/Fix Flow",
```

Comment:
What logs/CR details are needed when this test fails? They should be added to the reporter config here to assist failure debugging in CI: 

https://github.com/openshift-kni/eco-gotests/blob/main/tests/cnf/ran/gitopsztp/internal/tsparams/ztpvars.go#L32-L70